examples/exex/minimal/src/main.rs

```diff
@@ -50,3 +51,45 @@ fn main() -> eyre::Result<()> {
         handle.wait_for_node_exit().await
     })
 }
+
+#[cfg(test)]
+mod tests {
+    use std::{pin::pin, sync::Arc};
+
+    use reth::providers::{BundleStateWithReceipts, Chain};
+    use reth_exex::{ExExEvent, ExExNotification};
+    use reth_exex_test_utils::{test_exex_context, PollOnce, TestExExContext};
+
+    #[tokio::test]
```

Comment:
this should likely use multi-thread flavor

Reply:
unsure, why? we're not running the node